src/main/java/edu/nps/moves/dis/SignalPdu.java

```diff
@@ -32,6 +32,15 @@ public class SignalPdu extends RadioCommunicationsFamilyPdu implements Serializa
      */
     protected int encodingScheme;
 
+    /**
+     * encoding class: bits 14 - 15 of encodingScheme
+     */
+    protected int encodingClass = 0;
+
+    /**
+     * encoding type or number of TDL messages: bits 0 - 13 of encodingScheme
+     */
+    protected int encodingType = 0;
```

Comment:
Suggest we remove the encodingClass and encodingType data members and instead when we use the new setters and getters we work directly on the encodingScheme data member. This way the data isn't in two places. 

Unless there's a concern with performance, that's my suggestion.